index.bs

```diff
@@ -480,36 +480,16 @@ starts at 0 the first time {{getCurrentFrameData()}} is invoked for a given
 {{VRDisplay}}.
 
 <dfn attribute for="VRFrameData">leftProjectionMatrix</dfn>
-A 4x4 matrix describing the projection to be used for the left eye's rendering,
-represented as a 16 element array in column major order. This value may be
-passed directly to WebGL's uniformMatrix4fv function. It is highly recommended
-that applications use this matrix without modification. Failure to use this
-projection matrix when rendering may cause the presented frame to be distorted
-or badly aligned, resulting in varying degrees of user discomfort.
+A 4x4 matrix describing the projection to be used for the left eye's rendering, given as a 16 element array in column major order. This value may be passed directly to WebGL's uniformMatrix4fv function. It is highly recommended that applications use this matrix without modification. Failure to use this projection matrix when rendering may cause the presented frame to be distorted or badly aligned, resulting in varying degrees of user discomfort.
```

Comment:
Saying "column major" identifies the storage, but not the convention of the matrix. OpenGL uses column vectors that get multiplied from the right, whereas D3D convention is (usually) row vectors that get multiplied from the left. The only difference is a transpose, but we should point out that these are OpenGL style matrices where vectors are columns that get multiplied from the right.